documentation/assemblies/metrics/assembly_metrics-cruise-control.adoc

```diff
@@ -0,0 +1,69 @@
+// This assembly is included in the following assemblies:
+//
+// metrics/assembly-metrics.adoc
+
+[id='assembly-cruise-control-{context}']
+= Monitor Cruise Control
+
+If you are already using Prometheus and Grafana for monitoring of built-in Kafka metrics, you can configure Prometheus to also scrape the Cruise Control Prometheus endpoint.
+
+The example Grafana dashboard for the Cruise Control provides:
+
+* Information about optimization proposals computation, goals violation, cluster balancedness and more
+* Information about REST API calls for rebalance proposals and actual rebalance operations
+* JVM metrics from Cruise Control itself
+
+== Configuring Cruise Control
+
+You can enable the Cruise Control metrics in the `Kafka` resource using the `cruiseControl.metrics` property that contains the JMX exporter configuration about metrics to expose.
+
+For example:
++
```

Comment:
You don't need a plus here unless I'm mistaken!

Reply:
I copied it from the assembly_metrics-kafka-bridge.adoc, if you think it's not needed I am going to remove it anyway.